workspaces/jenkins/.changeset/twenty-houses-sparkle.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage-community/plugin-scaffolder-backend-module-jenkins': minor
```

Comment:
Right now this is a breaking change, as the package no-longer exports

https://github.com/backstage/community-plugins/pull/2134/files#diff-542602af3468f8d6d6a30d1a415df2136c98d435ac8facb84a08cd4b24c8e29cL11

Either we need to keep backwards compatibility, or you should call this out in the changeset that this package is now only compatible with the new frontend system.

Happy for the latter, it just needs to be called out as such.

Reply:
I would mark as breaking changes if is fine for you, consider which the change impact only one existing action.

So should I changed minor with major and update changeset description?